src/elements/datepicker/datepicker-toggle/datepicker-toggle.spec.ts

```diff
@@ -181,4 +183,76 @@ describe(`sbb-datepicker-toggle`, () => {
     expect(changeSpy.count).to.be.equal(1);
     expect(blurSpy.count).to.be.equal(1);
   });
+
+  it('handles view property', async () => {
```

Comment:
Is there a reason that made you test the view progression in the `datepicker-toggle` and the simple "opens on year/month view" in the calendar tests? My gut would tell me to do the other way around 😅

Reply:
It was the main use case and during manual testing, I had more problems there to get everything working. But I can think about it again.

Reply:
My idea was just that the whole selection mechanism working is just up to the calendar component, while the "only" thing the datepicker-toggle has to do is forward the property to the inner calendar. But once again that's just my gut feeling

Reply:
I remember now why I did those tests here.
one use case I can only test here: after closing the calendar by picking a date, it should re-open with the year view again. On the calendar, it just stays on the dayview. 
Furthermore, I could test, if the date of the calendar is still present in the calendar (sync).
The last part with changing to the month view is proving that passing down the change of the "view" property during the run time of the component works.

Reply:
Sounds convincing to me 👍